package.json

```diff
@@ -44,14 +39,19 @@
     "rtw <rtw1981@gmail.com>"
   ],
   "scripts": {
-    "cover": "nyc --reporter=lcov mocha --require intelli-espower-loader test/*.js && nyc report",
+    "cover":
```

Comment:
Looks like my prettier plugin ran on this JSON file. Harmless, but I might submit a followup to change it back.